src/components/Arguments/components.tsx

```diff
@@ -244,18 +244,20 @@ export const ActionButton: React.FC<InteractionButtonProps> = ({
   active = true,
   onClick,
 }) => {
-  const { project, active: activeEditor } = useProject();
+  const {
+    project,
+    active: activeEditor,
+    getActiveCode,
+    showSavingMessage,
+  } = useProject();
   const label = getLabel(type, project, activeEditor);
-  const { isSavingCode } = useProject();
-  const sendingTransaction = false;
-
+  const code = getActiveCode()[0].trim();
```

Comment:
will this fatal if getActiveCode() returns undefined?

Reply:
getActiveCode is typed as a [string, number] tuple so it seems like that shouldn't happen. However, since typescript assumes accessing an array always returns a value, it would technically be possible. I updated the default values, and we should consider setting noUncheckedIndexedAccess: true in the future